kms/capi/capi.go

```diff
@@ -941,7 +962,17 @@ func (s *CAPISigner) Sign(_ io.Reader, digest []byte, opts crypto.SignerOpts) ([
 		if !ok {
 			return nil, fmt.Errorf("unsupported RSA hash algorithm %v", hf)
 		}
-		signatureBytes, err := nCryptSignHash(s.keyHandle, digest, hashAlg)
+
+		var saltLength int
+		if rsaOpts, ok := opts.(*rsa.PSSOptions); ok {
+			if rsaOpts.SaltLength == rsa.PSSSaltLengthEqualsHash {
+				rsaOpts.SaltLength = rsaOpts.Hash.Size()
+			}
```

Comment:
Not cryptical, I can do this later, but when SaltLength is `rsa.PSSSaltLengthAuto` the salt length is:
```
(modulus_key_size - 1 + 7)/8 - hash_length - 2
```

For example, for a 4096-bit modules key and SHA256 it should be:
```
(4096 - 1 + 7)/8 - 32 - 2 = 478
```
See https://golang.org/cl/302230